### DISCUSSION TITLE
Error on Deforum by using Parseq

### DISCUSSION THREAD

**Discussion by AdrianYiu** [rewbs/sd-parseq#176] [Q&A]
Error: 'render_animation() takes 7 positional arguments but 8 were given'. Check your schedules/ init values please. Also make sure you don't have a backwards slash in any of your PATHs - use / instead of \. Full error message is in your terminal/ cli.

Anyone know what is this? How can I solve that? Thank you so much!

**Comment by KewkLW** [comment]
Link your settings file
